QueryBuilder/AnonymizedQueryBuilder.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the ekino/data-protection-bundle project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Ekino\DataProtectionBundle\QueryBuilder;
+
+use Ekino\DataProtectionBundle\Annotations\AnonymizedProperty;
+use Ekino\DataProtectionBundle\Meta\AnonymizedMetadata;
+
+/**
+ * Class AnonymizedQueryBuilder.
+ * This class aims to build an anonymizing query based on an AnonymizedMetadata object.
+ *
+ * @author Benoit Mazière <benoit.maziere@ekino.com>
+ */
+final class AnonymizedQueryBuilder
+{
+    /**
+     * return string
+     */
+    public function buildQuery(AnonymizedMetadata $anonymizedMetadata): string
+    {
+        $query = '';
+
+        if ($anonymizedMetadata->getAnonymizedEntity()->isTruncateAction()) {
+            $query = $this->buildTruncateQuery($anonymizedMetadata);
+        } elseif ($anonymizedMetadata->getAnonymizedEntity()->isAnonymizeAction()) {
+            $query = $this->buildAnonymizeQuery($anonymizedMetadata);
+        }
+
+        $exceptWhereClause = $anonymizedMetadata->getAnonymizedEntity()->getExceptWhereClause() ?? '';
```

Comment:
`getExceptWhereClause()` returns `string|null`. I don't see the value of `?? ''`.